src/components/PostEditor/PostEditor.connector.js

```diff
@@ -113,7 +109,6 @@ export const mapDispatchToProps = (dispatch) => {
     ...bindActionCreators({
       changeQueryString: replace,
       goToUrl: push,
-      fetchDefaultTopics,
```

Comment:
how come we dont need fetchDefaultTopics anymore?

Reply:
The answer to that is complicated and was too far back for me to summarize effectively now. I think this is mostly a naming thing though. Most/all Topic fetching actions/queries were moved to `store/actions`, and default may simply now be `fetchTopics` with appropriate params?